internal/pkg/cli/job_init.go

```diff
@@ -71,8 +71,7 @@ type initJobOpts struct {
 	manifestPath string
 	platform     *manifest.PlatformString
 
-	// Cache variables
-	wsAppName string
+	wsAppName *string // Cache variable for local app name. Empty if there's no app. Nil if pending creation.
```

Comment:
I see we want to be able to communicate this message "the workspace will have been created by the time `Execute` is run; have faith in it and don't validate".

What do you think of introducing a var named "pendingWSCreation" instead of aggregating the state by introducing a pointer? I have no preference but the first option mayyy seem clearer.

Reply:
The reason why i use the same variable has another reason: if we make it two variables then we set either one of them for initiation whereas if we keep one then whenever we init the opts we always set `wsAppName`. But I agree this is not exactly the same as our manifest pointer field as the `nil` value doesn't really make sense without further explanation, which harms the readability